dali/test/python/test_dali_tf_dataset_mnist.py

```diff
@@ -76,18 +76,18 @@ def define_graph(self):
 def _get_mnist_dataset(device='cpu', device_id=0, shard_id=0, num_shards=1):
     mnist_pipeline = MnistPipeline(
         4, data_path, device, device_id, shard_id, num_shards)
-    shapes = [
-        (BATCH_SIZE, IMAGE_SIZE, IMAGE_SIZE),
-        (BATCH_SIZE)]
-    dtypes = [
+    shapes = (
+        (64, 24, 24),
```

Comment:
So you can remove IMAGE_SIZE, but I would stick to usage of BATCH_SIZE here as it is used in other places and needs to match.

Reply:
This is leftover from testing, will revert.